contrib/util/express.php

```diff
@@ -46,126 +65,159 @@
  
     //setup of database
 
-    echo "Connecting to MySQL Server...\n";
+    echo "Connecting to MySQL Server...
```

Comment:
Oh no. I think that may have been another case of my autoformatter (whatever vscode extension) abusing its power again. I don't think phpcbf did that, but it may have. It would be based on our rules if so, which right now are pretty lacking in the PSR2 department.